desktop/hardened-desktop/registry-access-management.md

```diff
@@ -43,18 +43,20 @@ To configure Registry Access Management permissions:
    >
    > Once you add a registry, it takes up to 24 hours for the changes to be enforced on your developers’ machines. If you want to apply the changes sooner, you must force a Docker logout on your developers’ machine and have the developers re-authenticate for Docker Desktop.
 
-![Registry Access Management](../../docker-hub/images/registry-access-management.png){:width="700px"}
+> **Tip**
+>
+> Since RAM sets policies about where content can be fetched from, the [ADD](/engine/reference/builder/#add) instruction of the Dockerfile, when the parameter of the ADD instruction is a URL, is also subject to registry restrictions. It is recommended that you add the domains of URL parameters to the list of allowed registry addresses under the Registry Access Management settings of your organization.
```

Comment:
```suggestion
> Since RAM sets policies about where content can be fetched from, the [ADD](/engine/reference/builder/#add) instruction of the Dockerfile, when the parameter of the ADD instruction is a URL, is also subject to registry restrictions. It's recommended that you add the domains of URL parameters to the list of allowed registry addresses under the Registry Access Management settings of your organization.
```